libraries/botbuilder-dialogs/src/dialogSet.ts

```diff
@@ -23,33 +23,34 @@ import { DialogContext, DialogState } from './dialogContext';
  * for their name and phone number:
  *
  * ```JavaScript
- * const { DialogSet, TextPrompt } = require('botbuilder-dialogs');
+ * const { DialogSet, TextPrompt, WaterfallDialog } = require('botbuilder-dialogs');
  *
  * const dialogs = new DialogSet();
  *
- * dialogs.add('fillProfile', [
- *     async function (dc, options) {
+ * dialogs.add(new WaterfallDialog('fillProfile', [
+ *     async (dc, step) => {
  *         dc.activeDialog.state.profile = {};
- *         await dc.prompt('textPrompt', `What's your name?`);
+ *         return await dc.prompt('textPrompt', `What's your name?`);
  *     },
- *     async function (dc, name) {
- *         dc.activeDialog.state.profile.name = name;
- *         await dc.prompt('textPrompt', `What's your phone number?`);
+ *     async (dc, step) => {
+ *         dc.activeDialog.state.profile.name = step.results;
```

Comment:
The `dc.activeDialog.state` should now be `step.values` … so this line should be updated to `step.values.profile.name = step.result.`  … oh, and, it should singular form "step.result".

Reply:
I updated this.